src/app/dim-ui/ClosableContainer.m.scss

```diff
@@ -18,6 +18,9 @@
   &:hover {
     background-color: $orange;
   }
+  &:focus {
+    outline: 1px solid $blue;
```

Comment:
I just picked blue as it is closer to browser defaults but it doesn't look too bad. We already use orange for selected so I wasn't going to use that.